programs/uxd/src/utils/math/mod.rs

```diff
@@ -1,5 +1,4 @@
 pub mod checked_add_u128_and_i128;
-pub mod checked_convert_u128_to_u64;
```

Comment:
This function was created to do exactly what the maths function are doing now. Removing duplicate and using the `maths.rs` version instead